### ISSUE TITLE
Round BPM to nearest half-beat

### ISSUE THREAD

**Issue by DoubleDee73** [rakuri255/UltraSinger#128]
For most modern songs, it is highly unlikely, that they have been recorded without metronome or drum computer. It is not very likely that musicians or producers would deliverately set the BPM to something with decimals. 
My suggestion would therefore be, when you create a new song to round the BPM to the nearest half-integer before doubling/tripling/quadrupling the BPM.

**Comment by bohning** [comment]
I don't think that tripling was ever a thing, it is rather (potentially repeated) doubling.

**Comment by DoubleDee73** [comment]
I've seen it a couple of times. e. g. just stumbled across USDB ID 6308 where the BPM is 324. The actual BPM is 108, so I suppose, somebody tripled it. But doesn't matter, rounding it will do.

**Comment by rakuri255** [comment]
duplicate: #29
